palamar/settings.py

```diff
@@ -231,11 +232,14 @@
     'social_core.pipeline.social_auth.associate_by_email',
 )
 
+# Google auth settings
+
 # SECURITY WARNING: keep the secret key used in production secret!
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = '58128956219-lobtqsd956c9cpqvsd5ml2hvaa35ps9o'
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = 'RNXY2W32PhAsGVZKx8WsxTmK'
```

Comment:
I'll revoke all local keys immediately. After splitting setting file please ask me for new keys.